Mathlib/LinearAlgebra/BilinearForm/Properties.lean

```diff
@@ -185,6 +185,11 @@ theorem isAlt_neg {B : BilinForm R₁ M₁} : (-B).IsAlt ↔ B.IsAlt :=
   ⟨fun h => neg_neg B ▸ h.neg, IsAlt.neg⟩
 #align bilin_form.is_alt_neg LinearMap.BilinForm.isAlt_neg
 
+theorem Alt_eq_of_sum_zero {B : BilinForm R₁ M₁} {a b c : M₁} (hB : B.IsAlt) (hSum : a + b + c = 0) :
```

Comment:
```suggestion
theorem isAlt.eq_of_add_add_eq_zero {B : BilinForm R₁ M₁} {a b c : M₁} (hB : B.IsAlt) (hSum : a + b + c = 0) :
```
which means this can be use as `hB.eq_of_add_add_eq_zero` at the caller.

Note that `sum` is reserved for things like `List.sum` or `Finset.sum`, and we use `add` for expressions with `+`.

Reply:
In fact, if you move this above the `end IsAlt`, then you can drop the `IsAlt` prefix as it is added automatically